src/components/CountryDetail.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable no-undef */
+/* eslint-disable react/prop-types */
+import { FaArrowAltCircleRight } from 'react-icons/fa';
+import { useNavigate } from 'react-router-dom';
+import style from './components.module.css';
+
+function CountryDetail(props) {
```

Comment:
- [ ] Your app should respect the Layout: composition and space between elements, kindly make sure to use the same spacing as in the given design, also the info about the country should be at the bottom of the country flag

|Your design|Original design|
|-|-|
|![Screenshot_20220916_200645](https://user-images.githubusercontent.com/25227022/190713514-0ed6927a-088f-4ea8-81d9-a91448a5b28b.png)|![Screenshot_20220916_200827](https://user-images.githubusercontent.com/25227022/190713526-31d63d66-15cb-4a2f-b97e-a4e64d27ee97.png)|